util/task.py

```diff
@@ -84,7 +91,7 @@ def any_submitted(user_id, year_id):
         group_by(model.Task).all()
 
 
-def after_deadline():
+def after_deadline() -> set[int]:
```

Comment:
... `from typing import Set`

```suggestion
def after_deadline() -> Set[int]:
```
Navrhuji se držet staršího způsobu typování, kvůli kompatibilitě s verzí Pythonu na backendu

Reply:
A stejně tka prosím všechny výskyty `set`